### ISSUE TITLE
[MSOURCES-128] Build failure - Failed to read artifact

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-source-plugin#84]
**[Luigi Proietto](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=luigiproietto)** opened **[MSOURCES-128](https://issues.apache.org/jira/browse/MSOURCES-128?redirect=false)** and commented

Building EMP Connctor 0.01 - SNAPSHOT, it failed.

It could not transfer artifact org.apache.maven.plugins:maven-source-plugin:pom: 3.0.1 from/to central.

---

**Affects:** 3.0.1

**Attachments:**
- [IMG_20200401_201403.jpg](https://issues.apache.org/jira/secure/attachment/12998507/IMG_20200401_201403.jpg) (_3.26 MB_)

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

Please read
blog.sonatype.com/enhancing-ssl-security-and-http/2-support-for-central
or
https://stackoverflow.com/questions/50824789/why-am-i-getting-received-fatal-alert-protocol-version-or-peer-not-authentic

TLDR: easiest solution is to use at least Java 8 to run Maven
